client/cortex.js

```diff
@@ -1,11 +1,55 @@
-// Im a javascript file
-(function(window, $, _){
+//	Cortex.js
+
+//	(c) 2015 Sean McGary
+//	Cortex may be freely distributed under the MIT license.
+
+;(function(root, factory){
+	// Setup Cortex for the environment
+	//
+	// Done in a simliar way to Backbone https://github.com/jashkenas/backbone/blob/master/backbone.js
+	//
+	if(typeof define === 'function' && define.amd){
+
+		// Check for AMD
+		define(['lodash', 'backbone'], function(_, backbone){
+			root.Cortex = factory(root, _, backbone);
+			return root.Cortex;
+		});
+
+	} else if(typeof exports !== 'undefined'){
+
+		// Check for Node.js or CommonJS.
+		var _;
+		var lodash;
+		var underscore;
```

Comment:
Would be nice if a user could either require lodash or underscore declaratively with this. Why not use what's on Backbone._?